pom.xml

```diff
@@ -5,15 +5,16 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.40</version>
+    <version>4.80</version>
     <relativePath />
   </parent>
   <artifactId>no-agent-job-purge</artifactId>
-  <version>1.3-SNAPSHOT</version>
+  <version>${changelist}</version>
```

Comment:
Adopting incrementals requires adding a `.mvn` directory to enable `git-changelist-maven-extension`.

Reply:
Thanks for the help @basil 
Added .mvn